chain/client/src/tests/cross_shard_tx.rs

```diff
@@ -441,14 +441,7 @@ fn test_cross_shard_tx_common(
         let observed_balances = Arc::new(RwLock::new(vec![]));
         let presumable_epoch = Arc::new(RwLock::new(0usize));
 
-        let mut balances_local = balances.write().unwrap();
-        let mut observed_balances_local = observed_balances.write().unwrap();
-        for i in 0..8 {
-            balances_local.push(1000 + 100 * i);
-            observed_balances_local.push(0);
-        }
-
-        let (genesis_block, conn, block_stats) = setup_mock_all_validators(
+        let (_, conn, block_stats) = setup_mock_all_validators(
```

Comment:
why can't we use genesis block returned by `setup_mock_all_validators` here? 🤔

Reply:
This is leftover of my attempt to simplify `setup_mock_all_validators`. It spilled over, but I think genesis block shouldn't be returned by it - it returns clients which are able to request this block anyway. I'll remove it completely in future PR